scripts/karma/config.js

```diff
@@ -45,12 +45,21 @@ module.exports = {
                 'env',
                 'react',
               ],
-              plugins: ['transform-class-properties', 'istanbul'],
+              plugins: ['transform-class-properties'],
             },
           },
         }, {
-          test: /\.tsx?$/,
+          test: /\.tsx$/,
           loader: 'ts-loader',
+        }, {
+          enforce: 'post',
+          test: /\.(js|ts)x?$/,
+          use: {
+            loader: 'istanbul-instrumenter-loader',
```

Comment:
My understanding is that we're using `istanbul-instrumenter-loader` now instead of the `istanbul` webpack plugin, is that correct? can we remove `istanbul` from the package.json or does the loader require it?

Reply:
`istanbul-instrumenter-loader` requires `istanbul`...like when create-react-app requires `node-sass` when you want to use sass.

We needed to switch it because we the istanbul webpack plugin does not work with typescript.